include/modem/modem_info.h

```diff
@@ -57,14 +48,21 @@ extern "C" {
 /** SNR offset value. */
 #define SNR_OFFSET_VAL 24
 
-/** Modem returns RSRP and RSRQ as index values which require
- * a conversion to dBm and dB respectively. See modem AT
- * command reference guide for more information.
+/** @brief Converts RSRP index value returned by the modem to dBm.
+ *
+ * See modem AT command reference guide for more information.
```

Comment:
I think I will add the formula from the AT command reference guide when I get the latest draft and the wording in there.